AppCenterDemoApp/Assets/Puppet/PuppetTransmission.cs

```diff
@@ -446,9 +450,19 @@ public void TrackEventStringPropertiesChildTransmission()
             else
             {
                 var propertyConfigurator = transmissionTarget.GetPropertyConfigurator();
-                foreach (var property in properties)
+                if (properties != null)
                 {
-                    propertyConfigurator.SetEventProperty(property.Key, property.Value);
+                    foreach (var property in properties)
+                    {
+                        propertyConfigurator.SetEventProperty(property.Key, property.Value);
+                    }
+                }
+                if (propertiesParents != null)
+                {
+                    foreach (var property in propertiesParents)
```

Comment:
1DS supports inheriting from parent, here you are doing it in the app and not using the SDK mechanism.

Thus I don't really understand these changes. What am I missing?

Reply:
After talking to @annakocheshkova we believe this is not the right fix.
Native SDK has inheritance built in, please make sure that instead the child target is correctly created from parent and inheriting.